### DISCUSSION TITLE
sympy constants

### DISCUSSION THREAD

**Discussion by Peter230655** [sympy/sympy#25021] [Q&A]
I was told by Timo some time ago, that when I give a point zero speed, I should use 0, not 0., even though speed is normally not an integer.

Somewhere I read, that constants, say, 3 or 4.78 or... in sympy should be written as sympy.S(constant)
In programs written by much better programmers than me, I have seem both.

My question: what is the correct way of doing it?

Thanks for any help!

**Comment by Arniiiii** [comment]
https://docs.sympy.org/latest/modules/physics/units/quantities.html
